polaris-service/src/main/java/org/apache/polaris/service/BootstrapRealmsCommand.java

```diff
@@ -49,22 +48,24 @@ protected void run(
     MetaStoreManagerFactory metaStoreManagerFactory =
         configuration.findService(MetaStoreManagerFactory.class);
 
-    PolarisConfigurationStore configurationStore =
-        configuration.findService(PolarisConfigurationStore.class);
-
     // Execute the bootstrap
     Map<String, PrincipalSecretsResult> results =
         metaStoreManagerFactory.bootstrapRealms(configuration.getDefaultRealms());
 
     // Log any errors:
     boolean success = true;
     for (Map.Entry<String, PrincipalSecretsResult> result : results.entrySet()) {
+      var realmId = result.getKey();
+      var secretsResult = result.getValue();
       if (!result.getValue().isSuccess()) {
         LOGGER.error(
-            "Bootstrapping `{}` failed: {}",
-            result.getKey(),
-            result.getValue().getReturnStatus().toString());
+            "Bootstrapping `{}` failed: {}", realmId, secretsResult.getReturnStatus().toString());
         success = false;
+      } else {
+        var principalSecrets = secretsResult.getPrincipalSecrets();
+        System.out.printf(
+            "Bootstrap realm '%s' - root principal credentials: %s secret: %s",
```

Comment:
Also, IIRC, secrets are not stored in EclipseList anyway and the in-memory persistence prints the generated secrets in `InMemoryPolarisMetaStoreManagerFactory` (end of file).

I tend to think this printout is not necessary.

Reply:
We need this printout for the EL variant. The in-memory thing doesn't even have/need an explicit bootstrap - all realms are implicitly created, as many as you want (it's not intuitive and I think that behavioral difference should really be changed).

Reply:
> secrets are not stored in EclipseLink anyway 

The problem is, that it is possible - and before this change the clear text secrets were passed down to EL and therefore the database, which is IMHO an absolute no-go. But even if it was not the case, the fact that it _could_ happen is not good.

Reply:
Yes, I agree that it's a good idea to remove secret fields from JPA classes

Reply:
BTW: I've intentionally chosen `System.out` over `LOGGER` - it's not yet great though.

Reply:
> We need this printout for the EL variant.

EL bootstrapping is already impossible without env. vars. Isn't is what #461 attempts to fix?

I believe the discussion there was moving away from printouts. I think it would be safer to remove secret JPA fields first (this does not make the bootstrapping situation worse) and then deal with the bootstrap command in a separate PR. (My preference would be writing generated secrets to a file.)